src/providers/formattingProvider.ts

```diff
@@ -179,7 +179,7 @@ export class FormatProvider implements vscode.DocumentFormattingEditProvider {
 
             // return or ExitApp
             if (
-                purifiedLine.match(/\b(return|ExitApp)\b/i) &&
```

Comment:
If it's case-sensitive, it won't match when user's input `ExitApp` anymore, where it did the previous code. Since AutoHotkey is generally case-insensitive, we should be case-insensitive as well, right?

Reply:
Line 83:
`const purifiedLine = CodeUtil.purify(originalLine.toLowerCase());`
All text in `purifiedLine` is lower cased, all regexp(s) (from original extension's author) are wrote in lower case and case-sensitive, except that one.

I think all regexp must be or case-sensitive (wrote in lower case) or case-insensitive (wrote with CamelCase).

What do you think? Or just leave it like that :).

Reply:
Yes, if all text sent in is lower-cased, then we're good. I missed this one. As long as we have tests for this I'm happy with it :)